### PR TITLE
Add some documentation

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Added links to discord.js and dotenv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,7 @@ To get started working with this project locally you need to have node installed
 npm run dev
 ```
 
+## Documentation
+- [Discord.js](https://discord.js.org/#/docs/main/stable/general/welcome)
+- [dotenv](https://www.npmjs.com/package/dotenv)
```
Comment (bradtaniguchi) [inline README.md]: I don't think we need to link docs for dotenv. It should "just work" without us doing much besides setting up the 1 call (#2).

Comment (naomi-lgbt) [reply README.md]: Agreed and removed.

**Comment by naomi-lgbt** [comment]
Per #8, added disclaimer to README

**Review by bradtaniguchi — APPROVED** [review]
Nice, LGTM! 👍 :shipit:

**Pull request merged**